src/main/java/de/tum/in/www1/artemis/repository/StatisticsRepository.java

```diff
@@ -45,6 +69,16 @@
             """)
     List<Map<String, Object>> getReleasedExercises(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate);
 
+    @Query("""
+            select e.releaseDate as day, count(e.id) as amount
+            from Exercise e
+            where e.releaseDate >= :#{#startDate} and e.releaseDate <= :#{#endDate} and e.id in :exerciseIds
+            group by e.releaseDate
+            order by e.releaseDate asc
+            """)
+    List<Map<String, Object>> getReleasedExercisesForCourse(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate,
```

Comment:
please use a custom object here and not a map

Reply:
Done